plugins/rbac-backend/src/service/plugin-endpoints.ts

```diff
@@ -108,9 +114,12 @@ export class PluginPermissionMetadataCollector {
           ];
         }
       } catch (err) {
-        if (!isError(err) || err.name !== 'NotFoundError') {
-          throw err;
+        if (isError(err) && err.name === 'NotFoundError') {
+          continue;
         }
+        this.logger.error(
+          `Failed to retrieve permission metadata for ${pluginId}. ${err}`,
+        );
```

Comment:
My only concern with logging the permission to the console is that it could create a lot of spam. If there are ten plugins that are loading into an RHDH instance that do not have permissions, then there will be ten log statements each time the `/api/permission/plugins/polices` endpoint is accessed.

Reply:
I don't see a lot of logs. This is because, for backstage-plugins, if a plugin doesn't have a well-known endpoint, the API returns a 404 error, and we skip logging. In the case of backstage-showcase, if a plugin doesn't have a well-known endpoint, the API returns HTML, which our code fails to parse, and we silently skip this error (as a workaround).

Reply:
Ideally, it would be nice to retrieve and cache the list of permissions, but we cannot be sure when all dynamic plugins have completed startup.